runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java

```diff
@@ -584,6 +585,24 @@ private void hop(boolean previous)
 				}
 			}
 
+			// Check world region if filter is enabled
+			if (config.quickHopRegionFilter() != RegionFilterMode.NONE) {
```

Comment:
I think it would be better to put this here: https://github.com/runelite/runelite/blob/master/runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java#L563 instead, since it's separate to the world type checking.